mount.go

```diff
@@ -257,7 +257,20 @@ func mountStorage(storage pb.Storage) error {
 		return err
 	}
 
-	return mount(storage.Source, storage.MountPoint, storage.Fstype, flags, options)
+	var fsType = storage.Fstype
+	if (storage.Driver == driverSCSIType || storage.Driver == driverBlkType) && strings.Contains(storage.Fstype, "bind") {
```

Comment:
This new block could be handled by a function call that accepts a `storage *pb.Storage`. That would be a little cleaner, and you could then unit-test this logic.